src/adapters/github.js

```diff
@@ -306,6 +308,12 @@ class GitHub extends PjaxAdapter {
     $.ajax(cfg)
       .done((data) => {
         if (path && path.indexOf('/git/trees') === 0 && data.truncated) {
+          const hugeRepos = this.store.get(STORE.HUGE_REPOS);
+          const repo = `${opts.repo.username}/${opts.repo.reponame}`;
+          if (undefined === hugeRepos[repo]) {
+            hugeRepos[repo] = true;
```

Comment:
My concern is this list will keep growing and local storage is finite. At some point (probably a long time, but still..), Octotree won't be able to save anything any more. Maybe should keep only the most 100 recent large repos?

Reply:
Good point, I'll handle that